docs/kb/laptop-dev-env.md

```diff
@@ -0,0 +1,224 @@
+# NovaCustom laptop development environment
+
+This document describes the process of configuring a development and debugging
+environment for NovaCustom laptops.
+
+The instruction is applicable for workstations running Fedora. If you're using
+a different distro, some commands (e.g. package installation) may be different.
+
+## Embedded Controller
+
+This section describes the development environment for EC firmware on NovaCustom
+devices. The Embedded Controller firmware is heavily based on System76 EC
```

Comment:
```suggestion
devices. The Embedded Controller firmware is heavily based on [System76 EC](https://github.com/system76/ec)
```

Providing a reference seems like a right thing to do.

Reply:
https://github.com/Dasharo/docs/compare/4e9ec5622c840b8a6d840a400efafdd5de3ee80f..6a19fdc3b9c3ed99815a29ce364e8e90543a5ac2